frontend/src/app/patients/PatientForm.tsx

```diff
@@ -121,38 +122,31 @@ const UPDATE_PATIENT = gql`
   }
 `;
 
-const Fieldset = (props) => (
-  <fieldset className="prime-fieldset">
-    <legend>
-      <h3>{props.legend}</h3>
-    </legend>
-    {props.children}
-  </fieldset>
-);
+interface Props {
+  activeFacilityId: string;
+  patientId?: string;
+  patient?: any; //TODO: TYPES
```

Comment:
Do we have any reusable TS definitions for patients, queues, tests, etc? I didn't see any. There are some utilities we can use to generate them from the graphql file but ours is pretty generic (mostly `string`) so I'm not sure if it would be a lot of help in its current state.